google-cloud-spanner/src/main/java/com/google/cloud/spanner/AsyncResultSetImpl.java

```diff
@@ -578,6 +615,10 @@ public ResultSetMetadata getMetadata() {
     return delegateResultSet.get().getMetadata();
   }
 
+  public boolean initiateStreaming(StreamMessageListener streamMessageListener) {
```

Comment:
nit: add an `@Override` and `@InternalApi` or make the method package-private (if possible).

Reply:
done